apps/demos/testing/common.test.js

```diff
@@ -291,9 +294,10 @@ const SKIPPED_TESTS = {
     'Scheduler',
     'PivotGrid',
   ];
-  const BROKEN_THIRD_PARTY_SCRIPTS_COMPONENT = [
-    'Map',
-  ];
+
+  const EXPECTED_CONSOLE_MESSAGES = {
```

Comment:
Please, move this constant to known-warning.json file
Also, it would be good if you guys could plan task with moving other consts to according files too (like, create `known-a11y-issues.json`, `known-screenshot-issues.json` etc)